packages/web-components/fast-components/src/anchored-region/anchored-region.stories.ts

```diff
@@ -19,74 +17,103 @@ addons.getChannel().addListener(STORY_RENDERED, (name: string) => {
+            .addEventListener("click", (e: MouseEvent) => {
+                if (
+                    togglesRegion?.getAttribute("horizontal-default-position") === "right"
+                ) {
+                    togglesRegion!.setAttribute("horizontal-default-position", "left");
+                } else {
+                    togglesRegion!.setAttribute("horizontal-default-position", "right");
+                }
             });
 
-        const regionScalingUpdate = document.getElementById(
-            "region-scaling-update"
-        ) as FoundationAnchoredRegion;
         document
-            .getElementById("viewport-scaling-update")!
-            .addEventListener("scroll", () => regionScalingUpdate.update());
+            .getElementById("toggle-positions-vertical")!
+            .addEventListener("click", (e: MouseEvent) => {
+                if (togglesRegion?.getAttribute("vertical-default-position") === "top") {
+                    togglesRegion!.setAttribute("vertical-default-position", "bottom");
+                } else {
+                    togglesRegion!.setAttribute("vertical-default-position", "top");
+                }
+            });
 
-        const regionScalingOffset = document.getElementById(
-            "region-scaling-offset"
-        ) as FoundationAnchoredRegion;
         document
-            .getElementById("viewport-scaling-offset")
-            ?.addEventListener("scroll", (e: MouseEvent) => {
-                const target = e.target as HTMLElement;
+            .getElementById("toggle-inset-vertical")!
+            .addEventListener("click", (e: MouseEvent) => {
+                if (togglesRegion?.getAttribute("vertical-inset") === "true") {
+                    togglesRegion!.setAttribute("vertical-inset", "false");
+                } else {
+                    togglesRegion!.setAttribute("vertical-inset", "true");
+                }
+            });
 
-                regionScalingOffset.updateAnchorOffset(
-                    scalingViewportPreviousXValue - target.scrollLeft,
-                    scalingViewportPreviousYValue - target.scrollTop
-                );
+        document
+            .getElementById("toggle-inset-horizontal")!
+            .addEventListener("click", (e: MouseEvent) => {
+                if (togglesRegion?.getAttribute("horizontal-inset") === "true") {
+                    togglesRegion!.setAttribute("horizontal-inset", "false");
+                } else {
+                    togglesRegion!.setAttribute("horizontal-inset", "true");
+                }
+            });
 
-                scalingViewportPreviousXValue = target.scrollLeft;
-                scalingViewportPreviousYValue = target.scrollTop;
+        document
+            .querySelectorAll("[id^=anchor-menu-many]")
+            .forEach((el: HTMLButtonElement) => {
+                el.addEventListener("click", (e: MouseEvent) => {
+                    const menuNum = el.id.substr(16, el.id.length - 16);
```

Comment:
nit: the substr length parameter isn't needed since it's just going to the end of the id.